python-recipes/recommendation-systems/01_collaborative_filtering.ipynb

```diff
@@ -25,24 +25,54 @@
     "To generate our vectors we'll use the popular Python package [Surprise](https://surpriselib.com/)"
```

Comment:
Can we make the environment setup spiel identical between these? Install python deps, then redis setup, then data prep/setup? The first notebook shares a list of ways to get redis beyond in-notebook redis. I think we need to use that consistent markdown piece for all of the notebooks here!

Reply:
Sure thing